ui/src/app/applications/components/utils.tsx

```diff
@@ -753,7 +753,11 @@ export function syncStatusMessage(app: appModels.Application) {
         if (source.chart) {
             message += ' (' + revision + ')';
         } else if (revision.length >= 7 && !revision.startsWith(source.targetRevision)) {
-            message += ' (' + revision.substr(0, 7) + ')';
+            if(source.repoURL.startsWith("oci://")) {
+                message += ' (' + revision.substring(0, 14) + ')';
+            } else {
+                message += ' (' + revision.substring(0, 7) + ')';
+            }
         }
     }
 
```

Comment:
Is it possible that the `<Revision>` field in the lines below need to take into account the format of the `revision` value as well?

Reply:
Possibly - do you have any formatting in mind? Would you want the `sha256` prefix to be removed?